c/tskit/trees.h

```diff
@@ -36,22 +36,22 @@ extern "C" {
 
 #include <tskit/tables.h>
 
```

Comment:
Let's just put a ``// clang-format off`` around all the macro definitions here. It's handy having alignment, makes it more readable. (I guess we can put in a note saying that clang-format 9 has the option we want?)

Reply:
Also fixed in https://github.com/tskit-dev/tskit/pull/482/commits/37c7f83c4568e6444d9a102de4f00580775674da

I've left a note in `.clang-format` for when we upgrade.

Reply:
Great, thanks. Can we do the same for the error macros in ``core.h`` please?

Nearly there...

Reply:
Done